assets/scss/color-mixer.scss

```diff
@@ -1,58 +1,148 @@
+@each $name, $colour in $page-colours {
+
+  .page-colour--#{$name} {
+	.nhsuk-header, .nhsuk-header--white .nhsuk-search__submit, .nhsuk-header__navigation, #secondary section h2.widget-title, .nhsuk-panel-with-label__label, #secondary section #wp-calendar caption {
+	  background-color : $colour;
+	  background       : $colour;
+	}
+
+	a {
+	  color : $colour;
+	}
+
+	.nhsuk-header__navigation-link:hover, .nhsuk-header .nhsuk-header__navigation ul li a:hover {
+	  background-color : $colour;
+	  background       : $colour;
+	  filter           : brightness(85%);
+	}
+
+	.nhsuk-footer,
+	#secondary section {
+	  border-top-color : $colour;
+	}
+
+	@media (max-width : 40.0525em) {
+	  .nhsuk-header__navigation {
+		background       : $color_nhsuk-white;
+		background-color : $color_nhsuk-white;
+	  }
```

Comment:
Should this be inside the loop? Or should a default exist outside of this file?

Reply:
haha - was actually just looking at this. I am thinking if we make the mobile navigation default to white it slims the css, and also the hover colour on navigation, we can set that as a global 85% brightness and again that slims down the output and is more sustainable